src/components/color-picker/helpers/checkboard.js

```diff
@@ -5,7 +5,9 @@ export function render(c1, c2, size, serverCanvas) {
     return null;
   }
   const canvas = serverCanvas
-    ? new serverCanvas()
+    ? // todo : fix the error "A constructor name should not start with a lowercase letter"
```

Comment:
the solution here is to rename the variable to `ServerCanvas`

Reply:
fixed , thank you